### PR TITLE
Add icon server on self host services

### PR DESCRIPTION
**Pull request by nolway**
Add icon server on self host services.

### CODE REVIEW

File: contrib/docker/docker-compose.prod.yaml [hunk 1]
```diff
@@ -38,6 +38,7 @@ services:
       JITSI_URL: $JITSI_URL
       JITSI_PRIVATE_MODE: "$JITSI_PRIVATE_MODE"
       PUSHER_URL: //pusher.${DOMAIN}
+      PUSHER_URL: //icon.${DOMAIN}
```
Comment (tabascoeye) [inline contrib/docker/docker-compose.prod.yaml:41]: typo? did you mean `ICON_URL`?

Comment (moufmouf) [reply contrib/docker/docker-compose.prod.yaml:41]: Ping @nolway

Comment (moufmouf) [reply contrib/docker/docker-compose.prod.yaml:41]: Ping @nolway The problem is still here it seems :)

Comment (moufmouf) [reply contrib/docker/docker-compose.prod.yaml:41]:
```suggestion
      ICON_URL: //icon.${DOMAIN}
```

**Pull request merged**